### PR TITLE
Modified dashboard openebs.ci

### PR DESCRIPTION
**Pull request by bhaskarhc**
screenshot from 2019-02-27 12-51-47

### CODE REVIEW

**Comment by kalyantt** [comment]
Font size of the applications in the table has to be the same across the table, need a bit more spacing between the logo and the text. 2- The badges text is in CAPS it should be all small letters. 3- The border is not needed for the grey rows in the table.

**Comment by chandankumar4** [comment]
@bhaskarhc Can you remove the inline css from all files!!

File: src/app/workload-dashboard/workload-dashboard.component.html [hunk 1]
```diff
@@ -1,596 +1,245 @@
...
+                          <h1 style="font-size: 18px;color: #354052">Live deployments of stateful applications</h1>
+                          <h2 style="font-size: 14px;color: #7F8FA4">Workloads using Openebs PVs</h2>
+                          <input type="text" id="myInput" onkeyup="myFunction()" placeholder="Search applications.." title="Type in a name">     
+                        </div>
+                      <div class="row">
+                        <table class="table " id="myTable">
+                            <tr class="headrow header"  >
+                              <th class="headtext">Applications</th>
+                              <th class="headtext"> Replicas</th>
+                              <th class="headtext"> OpenEbs V</th>
```
Comment (chandankumar4) [inline src/app/workload-dashboard/workload-dashboard.component.html]: OpenEbs V --> OpenEBS V

File: src/app/header/header.component.html [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
+            <ul class="navbar-nav mr-auto">
+              <li>
+                <span style="margin-top:30px;" class="header-text">{{header}}</span>
+              </li>
+            </ul>
+            <ul class="navbar-nav mr-right ">
+                <li class="my-2  my-lg-0 form-inline head-top-icon">
+                    <a style="margin-right:15px;" placement="bottom" ngbTooltip="Like Us!" href="https://github.com/openebs/openebs/stargazers" target="_blank" class="btn btn-sm btn-outline-light fas fa-heart head-icons"></a>
+                    <a style="margin-right:15px; margin-left:15px;" data-toggle="modal" data-target="#subscribeModal" placement="bottom" ngbTooltip="Subscribe to our news letter" class="btn btn-sm btn-outline-light fas fa-envelope-open-text head-icons"></a>
+                    <a style="margin-left:15px;" ngbTooltip="Join Community"  placement="left" href="https://github.com/openebs/openebs/stargazers"  class="btn btn-sm btn-outline-light fab fa-slack head-icons"></a>
```
Comment (chandankumar4) [inline src/app/header/header.component.html]: This link should be https://slack.openebs.io

File: src/app/workload-dashboard/workload-dashboard.component.html (hunk 1 above)
Comment (bhaskarhc) [reply src/app/workload-dashboard/workload-dashboard.component.html]: done :+1:

File: src/app/header/header.component.html (hunk 2 above)
Comment (bhaskarhc) [reply src/app/header/header.component.html]: Done :+1:

**Pull request merged**